backend/src/main/java/ba/unsa/etf/si/bbqms/domain/Tenant.java

```diff
@@ -0,0 +1,121 @@
+package ba.unsa.etf.si.bbqms.domain;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
```

Comment:
No unused imports. Check other files. You can configure IntelliJ to automatically remove these.

Reply:
Removed unused imports